libs/utils/fonts.js

```diff
@@ -18,7 +18,13 @@ export default function loadFonts(locale, loadStyle) {
   const tkSplit = locale.tk.split('.');
   if (tkSplit[1] === 'css') {
     return new Promise((resolve) => {
-      loadStyle(`https://use.typekit.net/${locale.tk}`, resolve);
+      const isSafari = navigator.userAgent.includes('Safari') && !navigator.userAgent.includes('Chrome');
+      const PHONE_SIZE = window.screen.width < 550 || window.screen.height < 550;
```

Comment:
Where does the 550px value come from? 🤔 Isn't mobile usually 600px

Reply:
Im not sure how personalization got the 550px value either 🤔

Reply:
I'd also be more comfortable having the standard `600` value as the mobile one. So depending on your findings for iPad, maybe we can standardize this with our media queries. Looks good otherwise!